redbot/cogs/mod/events.py

```diff
@@ -72,6 +71,60 @@ async def check_mention_spam(self, message):
                         print(e)
                         return False
                     return True
+
+        if mention_spam["kick"]:
+            if len(mentions) >= mention_spam["kick"]:
+                try:
+                    await guild.kick(author, reason=_("Mention Spam (Autokick)"))
+                except discord.HTTPException:
+                    log.info(
+                        "Failed to kick member for mention spam in server {}".format(guild.id)
```

Comment:
Should be "Failed to kick **a** member for...". Also maybe also include the member's ID in this log?